helix-term/src/config.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::{Error, Result};
+use std::{collections::HashMap, str::FromStr};
+
+use serde::{Deserialize, Serialize};
+
+use crate::keymap::{parse_keymaps, Keymaps};
+
+#[derive(Default)]
+pub struct Config {
+    pub keymaps: Keymaps,
+}
```

Comment:
Can we use `serde::Deserialize` for this rather than hand-roll our own? We could use a custom `keymap` deserializer.